py2plantuml.sh

```diff
@@ -19,9 +18,9 @@ while [[ "$1" != "" ]]; do
         shift;
         ;;
         -d | --dependency_install )
-        echo "Downloading plantuml java binary in the current dir from github......"
-        wget -O plantuml.jar https://github.com/plantuml/plantuml/releases/download/v1.2022.13/plantuml.jar
-        echo "Finished installing plantuml dependency for the script!!"
+        echo "Installing plantweb dependency for rendering SVC....."
+        pip3 install plantweb
+        echo "Finished installing plantweb dependency for the script!!"
         ;;                            
```

Comment:
Hi @MSarfraz-CR actually this could be done but we need to inform the user that all data is going to be sent over the internet on a plantweb server. So we would need both, one plantweb for people using non sensitive information and not having time to install plantuml and the current one expecting plantuml to be installed locally.

Reply:
Fixed